messages/subjects.yaml

```
@@ -121,11 +119,14 @@ point_cloud:                            foxglove.PointCloud
 alarm:                                  keelson.Alarm
 audio:                                  keelson.Audio
 
-imu_reading:                            keelson.ImuReading
 
-imu_acceleration_reading:               keelson.AcceelerationReading
-imu_gyroscope_reading:                  keelson.GyroscopeReading
-imu_magnetic_field_reading:             keelson.MagneticFieldReading
+linear_acceleration_mpss:               keelson.Decomposed3DVector
+angular_velocity_radps:                 keelson.Decomposed3DVector
+magnetic_field_gauss:                   keelson.Decomposed3DVector
+
+force_magnitude_newton:                 keelson.Decomposed3DVector
+moment_magnitude_newton_meter:          keelson.Decomposed3DVector
```

Remove the magnitude part (its not just the magnitude, its the direction also since its a vector)
```suggestion
force_newton:                 keelson.Decomposed3DVector
moment_newton_meter:          keelson.Decomposed3DVector
```